client/app/hosting/hosting.service.js

```diff
@@ -539,6 +539,13 @@
                 });
             }
 
+            retrievingDetails (serviceName) {
+                return this.OvhHttp
+                    .get(`/hosting/web/${serviceName}/ssl/report`, {
```

Comment:
@jleveugle Is it preferable to start using [ovh-api-services](https://github.com/ovh-ux/ovh-api-services)?

Reply:
It should be part of a complete refactoring for the web manager then